Adventure/Assets/Scripts/InteractiveObject.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
```

Comment:
Add a summary here so someone can understand what this script does.

Reply:
Added the summary